stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -93,10 +94,10 @@ class Process():
     def is_alive(self) -> bool: ...
     def join(self, timeout: Optional[float] = ...) -> None: ...
 
-class Queue():
+class Queue(queue.Queue):
```

Comment:
So this implicitly makes the base class `queue.Queue[Any]`. I wonder if it wouldn't be better to make this `Queue` class generic as well, so one can instantiate a `mp.Queue[int]`, say.